src/core/event/EventManager.js

```diff
@@ -0,0 +1,60 @@
+/* global console */
+
+export default class EventManager {
+  constructor(nextHandlers) {
```

Comment:
The name here throws me off, i.e. the `next` in `nextHandlers`? Is the idea that the app can have multiple handlers for the same event? Why not just call the parameter `handlers`?

Some JSDoc would be invaluable.